packages/html-imports/package.json

```diff
@@ -22,6 +22,7 @@
   "license": "BSD-3-Clause",
   "scripts": {
     "build": "gulp",
+    "build:watch": "chokidar 'src/**/*.js' -c 'npm run build'",
```

Comment:
AFAICT, chokidar doesn't do an initial run like `tsc --watch`, so the ones that use chokidar to watch the actual source (vs JS output from `tsc` that are written during the build) probably need to be prefixed with `npm run build && `.

I found this [`ignoreInitial` option](https://github.com/paulmillr/chokidar#path-filtering) - it reads like it's talking about this situation and should do what we want by default, but I'm not seeing that happen in my repo. Maybe I'm missing something?

Reply:
You want to look at https://github.com/kimmobrunfeldt/chokidar-cli for the flags, you might have been looking at the underlying library instead of the CLI. I added the `--initial` flag, which does this.

Reply:
SGTM